source/ps4.rst

```diff
@@ -0,0 +1,143 @@
+Switching your robot to using a PS4 controller
+==============================================
+
+This procedure guides you in updating your robot to be controlled
+with a PS4 controller.
+
+*Who is this guide intended for?:*
+
+- Customers who received a PS3 controller with their robot, and would like to instead use
+  a PS4 controller with their robot.
+
+*What requirements are there for using a PS4 controller?:*
+
+- Your robot needs to be running Ubuntu 18.04 + ROS Melodic
+- You will need a PS4 controller (also called *DualShock 4 Wireless Controller
+  for PlayStation 4*).
+
+.. warning:: Third party controllers may not work. Only Sony PS4 controllers have been tested.
+
+Upgrade your Fetch robot's packages
+-----------------------------------
+Run the following to upgrade packages that are relevant::
+
+   sudo apt-get update
+   sudo apt-get install --only-upgrade ros-indigo-*
```

Comment:
Shouldn't these be `apt`?

Reply:
apt vs apt-get doesn’t make a big difference but I do think it should be “-indigo-“

Reply:
s/indigo/melodic, yeah :+1: